### PR TITLE
Add hidden style to display

### PR DESCRIPTION
**Pull request by captainsafia**
Adding this to work towards output hiding in nteract the notebook.

I figured I would put it here as well so anyone using an output component anywhere has the ability to hide it. I'm having an issue running the tests for this so I haven't actually confirmed that it works.

### CODE REVIEW

**Comment by captainsafia** [comment]
Weird that Travis passed this when in reality the tests failed. Looking into this....

**Comment by captainsafia** [comment]
Ugh. This there is a really annoying bug in the `electron-mocha` where it fails to parse the arguments passed in the command line properly.

**Comment by rgbkrk** [comment]
This should be configured by users to set `display:none` by themselves rather than handling it in the component.

```jsx
<DisplayArea outputs={outputs} style={display: 'none'}>
```

Or they create their own component that "builds" on top of our little lego piece here:

```jsx
export default const TogglableDisplayArea = (props) => {
  const style = {display: this.props.hidden ? 'none' : 'block'}
  return <DisplayArea {...this.props} style={style}>
}
```

We should not include the logic into this component itself.

**Comment by captainsafia** [comment]
OK! I added a ToggleableDisplay!

File: src/ToggleableDisplay.jsx [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+import Immutable from 'immutable';
+
+import { transforms, displayOrder } from 'transformime-react';
+
+import Display from './Display';
+
+export default function ToggleableDisplay(props) {
+  const style = { display: props.isHidden ? 'none' : 'block' };
+  return (
+    <Display props={props} style={style}/>
```
Comment (captainsafia) [inline src/ToggleableDisplay.jsx]: Erp. Replace it with spread operator.

File: src/index.js [hunk 2]
```diff
@@ -1,2 +1,4 @@
 import Display from './Display';
-export default Display;
```
Comment (rgbkrk) [inline src/index.js]: I just noticed this is a major change to the API, we either have to update all code that uses this everywhere and call it 2.0 or export both the default and `TogglableDisplay`

**Pull request merged**